src/SprykerEco/Zed/AmazonQuicksight/Communication/Controller/UserController.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\AmazonQuicksight\Communication\Controller;
+
+use Spryker\Zed\Kernel\Communication\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Security\Csrf\CsrfToken;
+
+/**
+ * @method \SprykerEco\Zed\AmazonQuicksight\Persistence\AmazonQuicksightRepositoryInterface getRepository()
+ * @method \SprykerEco\Zed\AmazonQuicksight\Business\AmazonQuicksightFacadeInterface getFacade()
+ * @method \SprykerEco\Zed\AmazonQuicksight\Communication\AmazonQuicksightCommunicationFactory getFactory()
+ */
+class UserController extends AbstractController
+{
+    /**
+     * @var string
+     */
+    protected const PARAM_REFERER = 'referer';
+
+    /**
+     * @var string
+     */
+    protected const ERROR_MESSAGE_CSRF_TOKEN_INVALID = 'CSRF token is not valid.';
+
+    /**
+     * @var string
+     */
+    protected const SUCCESS_MESSAGE_USERS_SYNCHRONIZED = 'Analytics users have been successfully synchronized.';
+
+    /**
+     * @uses \SprykerEco\Zed\AmazonQuicksight\Business\Expander\AnalyticsExpander::FIELD_NAME_TOKEN
+     *
+     * @var string
+     */
+    protected const FIELD_NAME_TOKEN = '_token';
```

Comment:
Do we need to align the const name with the name we are using in AnalyticsExpander?